server_worker/server_worker.py

```diff
@@ -192,18 +192,19 @@ def import_data_by_ibis(
         compression_type="gzip",
         skiprows=None,
         validation=None,
+        use_columns_types_for_pd=True,
     ):
         "Import CSV files using Ibis load_data to the OmniSciDB from the Pandas.DataFrame"
 
-        if columns_types:
+        if columns_types and use_columns_types_for_pd:
             columns_types_pd = convert_type_ibis2pandas(columns_types)
```

Comment:
```suggestion
        columns_types_pd = convert_type_ibis2pandas(columns_types) if columns_types and use_columns_types_for_pd else None
```
and use it down below without extra `if`s